docker.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -x
+RUST_TOOLCHAIN=1.49
+# TODO: check if it's defined, otherwise use default:
+ARCH=x86_64
```

Comment:
Can we build for the arch that we run the script on? I don't think cross builds work.